### PR TITLE
ping maintainers in case a PR was not properly labeled

### PR DESCRIPTION
**Pull request by pmeier**
Addresses https://github.com/pytorch/vision/issues/3351#issuecomment-794418768.

### CODE REVIEW

File: .github/process_pr.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import json
+import sys
+from typing import Any, Dict, Set, Tuple
+
+import requests
+
+# If the PR has any of these labels, we mark it as properly labeled.
+REQUIRED_LABELS = {
+    "TODO",
```
Comment (pmeier) [inline .github/process_pr.py:9]: Any suggestions what should go in here?

**Review by pmeier — COMMENTED** [review]
The workflow will probably not work the first few times due to small bugs. Since I cannot test the workflow locally, we need to merge and fix the teething issues as we go. I don't expect false positives, i.e. noise on the PRs.

File: .github/process_pr.py (hunk 1 above)
Comment (NicolasHug) [reply .github/process_pr.py:9]: Thank @pmeier !

We'll clean them as we go but for now I can think of:

- all the "module" labels
- all the "topic" labels
- enhancement
- bug
- code quality
- deprecation
- windows


CC @fmassa @datumbox 

Also, I thought we had a "New feature" tag but I don't see it? Did someone remove it?

Comment (datumbox) [reply .github/process_pr.py:9]: +1 on having the `module`/`topic` labels as a "top level" label. If we do adopt this, we need to change the way our release script works. At least the one we used for the upcoming release notes, classified as "unlabelled" PRs that had module and topic labels.

Not sure about `windows`, but no strong opinions.

The `new feature` makes sense to me, though I would like us to review all labels and provide concrete definitions to be aligned on those PRs that are on the boarder of two categories.

Comment (NicolasHug) [reply .github/process_pr.py:9]: @datumbox and I just went over our list of tags and we concluded that we could be using the following set  here:
```py
REQUIRED_LABELS = {
    'new feature',
    'bug',
    'code quality',
    'enhancement',
    'bc-breaking',
    'dependency issue',
    'deprecation',
    'module: c++ frontend',
    'module: ci',
    'module: datasets',
    'module: documentation',
    'module: io',
    'module: models.quantization',
    'module: models',
    'module: onnx',
    'module: ops',
    'module: reference scripts',
    'module: rocm',
    'module: tests',
    'module: transforms',
    'module: utils',
    'module: video',
    'Perf',
    'Revert(ed)',
}
```

**Review by datumbox — APPROVED** [review]
Stamping so that you can merge when ready and test it out.

**Review by fmassa — COMMENTED** [review]
Nice, thanks for this @pmeier!

**Pull request merged**